### PR TITLE
changes to the remote Docker env

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/building-docker-images.md [hunk 1]
```diff
@@ -150,10 +141,12 @@ CircleCI supports multiple versions of Docker. The following are the available v
 Consult the [Stable releases](https://download.docker.com/linux/static/stable/x86_64/) or [Edge releases](https://download.docker.com/linux/static/edge/x86_64/) for the full list of supported versions.
 --->
 
-**Note:** The `version` key is not currently supported on CircleCI server installations. Contact your system administrator for information about the Docker version installed in your remote Docker environment.
+The `version` key is not currently supported on CircleCI server installations. Contact your system administrator for information about the Docker version installed in your remote Docker environment.
+{: class="alert alert-info"}
 
 ## Separation of environments
```
Comment (rosieyohannan) [inline jekyll/_cci2/building-docker-images.md:147]: @sebastian-lerner There's a whole section on separation of environments here but I don't the info to be able to know how it needs to be updated. Could you point me in the right direction?

Comment (sebastian-lerner) [reply jekyll/_cci2/building-docker-images.md:147]: @brentmmarks any idea if any of these subsections are still applicable? i imagine the first two arent?

Comment (brentmmarks) [reply jekyll/_cci2/building-docker-images.md:147]: think you are right

File: jekyll/_cci2/building-docker-images.md [hunk 2]
```diff
@@ -16,30 +16,21 @@ This page explains how to build Docker images for deployment and further testing
...
           version: 20.10.14
 ```
 
-When `setup_remote_docker` executes, a remote environment will be created, and your current [primary container]({{ site.baseurl }}/glossary/#primary-container) will be configured to use it. Then, any docker-related commands you use will be safely executed in this new environment.
-
-**Note:** The use of the `setup_remote_docker` key is reserved for configs in which your primary executor _is_ a docker container. If your executor is `machine` (and you want to use docker commands in your config) you do **not** need to use the `setup_remote_docker` key.
+The use of the `setup_remote_docker` key is reserved for configs in which your primary executor _is_ a docker container. If you are using an execution environment **other** than Docker, and you want to use docker commands in your config, you do **not** need to use the `setup_remote_docker` key.
+{: class="alert alert-info"}
 
 ## Specifications
```
Comment (rosieyohannan) [inline jekyll/_cci2/building-docker-images.md:35]: @brentmmarks @sebastian-lerner I guess these specs aren't correct anymore?

**Review by sebastian-lerner — COMMENTED** [review]
it might also be good to add a note that remote docker jobs show up in the UI as "executor = Remote Docker" just so that it's easy to distinguish between a job using remote docker and a job that doesn't? https://discuss.circleci.com/t/changes-to-remote-docker-reporting-pricing/47759/6

**Review by sebastian-lerner — APPROVED** [review]
i think this looks good! one small thing, it might be worth noting why the feature is called "remote docker" despite there not being anything "remote" anymore...maybe something that says it used to be a "remote" environment that executed docker commands?

ill defer to you as to whether we want to include that or not but the name is a bit confusing given we say in these docs that everything runs locally

File: jekyll/_cci2/building-docker-images.md [hunk 3]
```diff
@@ -1,27 +1,20 @@
...
+description: "How to build Docker images using the Docker execution environment"
 contentTags:
   platform:
   - Cloud
   - Server v4.x
   - Server v3.x
 ---
 
-This page explains how to build Docker images for deployment and further testing. The examples on this page that use the Docker execution environment show how to start services in a remote docker environment.
+This page explains how to build Docker images for deployment and further testing when using the Docker execution environment. This is achieved using the remote Docker environment (`setup_remote_docker`). When you use the remote Docker environment for a job, any Docker commands you run in your job will be executed locally on the virtual machine used to spin up your primary Docker container. The term _remote_ used here is left over from when remote Docker usage would spin up a separate, remote environment for running Docker commands.
```
[the remark this answers is not in the document]
Comment (sebastian-lerner) [reply jekyll/_cci2/building-docker-images.md:12]: yeah looks great thank you

**Review by kbatuigas — CHANGES_REQUESTED** [review]
Looks good! Just one correction and one suggestion

File: jekyll/_cci2/building-docker-images.md [hunk 4]
```diff
@@ -1,27 +1,20 @@
...
-<br>
-Visit the [Discuss post](https://discuss.circleci.com/t/setup-remote-docker-architecture-change/45303) to learn more about the new architecture, and to follow updates regarding the rollout.
-{: class="alert alert-info"}
-
-To build Docker images for deployment using the Docker execution environment, you must use a special `setup_remote_docker` key which creates a separate environment for each build for security. This environment is remote, fully-isolated and has been configured to execute Docker commands. If your job requires `docker` or `docker-compose` commands, add the `setup_remote_docker` step into your `.circleci/config.yml`:
+To build Docker images for deployment using the Docker execution environment, you must use a special `setup_remote_docker` key. If your job requires `docker` or `docker-compose` commands, add the `setup_remote_docker` step into your `.circleci/config.yml`:
```
Comment (rosieyohannan) [inline jekyll/_cci2/building-docker-images.md:17]:
```suggestion
To build Docker images for deployment using the Docker execution environment, you must use the `setup_remote_docker` key. If your job requires `docker` or `docker-compose` commands, add the `setup_remote_docker` step into your `.circleci/config.yml`:
```

Comment (rosieyohannan) [inline jekyll/_cci2/building-docker-images.md:17]:
```suggestion
To build Docker images for deployment using the Docker execution environment, you must use the `setup_remote_docker` key. If your job requires `docker` or `docker-compose` commands, add the `setup_remote_docker` step into your `.circleci/config.yml`:
```

**Review by kbatuigas — APPROVED** [review]
🚀

**Pull request merged**